Algorithms/Trees/list_to_bst.cpp

```diff
@@ -0,0 +1,102 @@
+/*  Here, n elements are inserted and O(n) time is taken
+    to insert each element in the BST. Hence, we
+    end up with an overall time complexity of O(n*n), worst case.
+
+    In general however, overall time complexity is O(n*h) where
+    h is the height of the tree.
+    Again, n = 2^( h + 1 ) - 1 for a tree.
```

Comment:
This is not true for all trees.

Reply:
True for complete binary search tree, worst case if tree happens to me skewed